src/audio/selector/selector.c

```diff
@@ -437,7 +481,7 @@ static int selector_prepare(struct comp_dev *dev)
 		goto err;
 	}
 
-	return PPL_STATUS_PATH_STOP;
```

Comment:
I assume we can no longer do these stops because of the parameter propagation?

Reply:
@lgirdwood @paulstelian97 
Yes. As I understand, now on master, selector component is hard coded for KPB topology (I think that was the only use). We have got all these ```PPL_STATUS_PATH_STOP``` in ```selector_prepare()```, ```selector_trigger()```, ```selector_reset()``` in order not to propagate them downstream(playback)/upstream(capture). I've also removed ```sourceb->channels = 1;``` in ```test_keywords_params()``` (line 403) - it was also hard coded for KPB topology.